tests/cypress/integration/case_4_assign_taks_job_users.js

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright (C) 2020 Intel Corporation
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+/// <reference types="cypress" />
+
+context('Multiple users. Assign task, job.', () => {
+    const caseId = '4'
+    const labelName = `Case ${caseId}`
+    const taskName = `New annotation task for ${labelName}`
+    const attrName = `Attr for ${labelName}`
+    const textDefaultValue = 'Some default value for type Text'
+    const image = `image_${labelName.replace(' ', '_').toLowerCase()}.png`
+    const width = 800
+    const height = 800
+    const posX = 10
+    const posY = 10
+    const color = 'gray'
+    const secondUserName = 'Seconduser'
+    const thirdUserName = 'Thirduser'
+
+    const secondUser = {
+        firstName: `${secondUserName} fitstname`,
+        lastName: `${secondUserName} lastname`,
+        emailAddr: `${secondUserName.toLowerCase()}@local.local`,
+        password: 'UfdU21!dds'
+    }
+    const thirdUser = {
+        firstName: `${thirdUserName} fitstname`,
+        lastName: `${thirdUserName} lastname`,
+        emailAddr: `${thirdUserName.toLowerCase()}@local.local`,
+        password: 'Fv5Df3#f55g'
+    }
+
+    function closeModal() {
+        cy.get('.ant-modal-body').within(() => {
+            cy.get('.ant-modal-confirm-title')
+            .should('contain', 'Unsupported platform detected')
+            cy.get('.ant-modal-confirm-btns')
+            .contains('OK')
+            .click()
+        })
+    }
+
+    describe(`Testing case "${caseId}"`, () => {
+        // First user is "admin".
```

Comment:
Is admin logged out before this test?

Reply:
Yes. Cypress clears cookies before each test. Therefore, have to login again before each test.

Conflicts resolved.